packages/react/src/components/AccountSettings/ChangePassword/ChangePassword.tsx

```diff
@@ -195,7 +196,7 @@ function ChangePassword({
           onChange={handleChange}
           validationErrors={validationError?.newPassword}
         />
-        <DefaultPasswordField
+        <ConfirmPassword
           autoComplete="new-password"
```

Comment:
Should be `"confirm-password"`?

Reply:
autocomplete should be `new-password` for confirm password since it's a new password: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete#values